blocks/quilted-image-block/_index.scss

```diff
@@ -0,0 +1,53 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-quilted-image {
+	@include scss.block-properties("quilted-image");
+	@include scss.block-components("quilted-image");
+}
+
+.b-quilted-image--wrapper {
+	display: grid;
+
+	.b-quilted-image--wrapper--bottom {
```

Comment:
add the mixins like to the --bottom and --top

```scss
@include scss.block-properties("quilted-image");
	@include scss.block-components("quilted-image");
```

Reply:
addressed!

Reply:
^^ Mentioned includes have been added to all block class selectors.